conf/celeryconfig.py

```diff
@@ -5,17 +5,27 @@
 from celery.schedules import crontab
 
 from chronam.settings import *  # NOQA
-from chronam.settings import INSTALLED_APPS
 
 APP_DIR = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
 
 
-CELERY_IMPORTS = ("chronam.core.tasks",)
-CELERYD_LOG_FILE = os.path.join("/logs/celery", "celery.log")
+CELERY_IMPORTS = ("chronam.core.tasks")
+CELERYD_LOG_FILE = os.path.join("/var/log/celery", "celery.log")
 CELERYD_LOG_LEVEL = logging.INFO
 CELERYD_CONCURRENCY = 2
 
 CELERYBEAT_SCHEDULE = {
+    "poll_cts": {
```

Comment:
This is duplicating the code below in the conditional section which only enables CTS support when requested.